### ISSUE TITLE
Feature: navigate from source code to translation message

### ISSUE THREAD

**Issue by rillig** [vslavik/poedit#319]
Use case:

In `mailutils-3.1.de.po`, message ID 1024 is:

```
msgid "crit"
msgstr "crit"
```

The corresponding source code is:

```
char *_mu_severity_str[] = {
  N_("debug"),
  N_("info"),
  N_("notice"),
  N_("warning"),
  N_("error"),
  N_("crit"),
  N_("alert"),
  N_("emerg"),
};
```

The thing is that several of these strings, e.g. `warning`, have a message ID far away from 1024, so that they are not visible on the current screen.

![2017-01-09 23_16_04-mailutils-3 1 de po gnu mailutils 3 1 geandert - poedit beta-version lauft](https://cloud.githubusercontent.com/assets/3233724/21785884/b19710da-d6c1-11e6-9bcc-12b74e67dcb5.png)

To quickly navigate to the corresponding message, I'd like to right-click on the source code line of `N_("warning")` and find a menu item called `Jump/Navigate to message`.

**Comment by vslavik** [comment]
I don’t think this is worth doing. The source code viewer is not much used and this requirement is even more rare (so is the situation in your screenshot where it makes some sense). You can accomplish the same by using *View → Sort by File Order* (which, if I’m not mistaken, is even the default).

That said, I would have no problem merging a good PR implementing it if you want to tackle it. But realistically speaking, I’m unlikely to do it myself.
